doc/man3/ISSUER_SIGN_TOOL_new.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-ISSUER_SIGN_TOOL_new, ISSUER_SIGN_TOOL_free,ISSUER_SIGN_TOOL_it,
```

Comment:
I think it would be just better to drop this manual page and add all these to the X509_dup.pod to avoid duplication. The manual page is just a generic one, there is no special info here.

Reply:
fix commited